examples/keras_spark3_rossmann.py

```diff
@@ -0,0 +1,567 @@
+# Copyright 2017 onwards, fast.ai, Inc.
```

Comment:
This 567 lines script deviates from `examples/keras_spark_rosspann.py` in only 11 rows. Differences should only be spark3 specific, but I suspect both files will diverge quite quickly on other changes.

I suggest to put some github actions in place that compare both scripts and flag up unexpected deviations, as put in place for `README.rst` and `docs/summary.rst`. I am happy to create a PR for this once this is merged into master.

Reply:
Good idea.  At some point we may want to further consolidate things so that we don't have so many nearly identical scripts.  Once Spark 3 is out, we may just replace the old examples or add special handling for older versions of Spark.